shelterapp/models.py

```diff
@@ -31,3 +31,7 @@ def __str__(self):
     # Add get_absolute_url method to Dog model
     def get_absolute_url(self):
         return reverse('dog_detail', kwargs={'pk': self.pk})
+
+class Owner(models.Model):
+    name = models.CharField(max_length=50)
+    email = models.CharField(max_length=50)
```

Comment:
```suggestion
    email = models.EmailField(max_length=50)
```

`EmailField` is a better choice for email addresses